### PR TITLE
[INTERNAL] Deprecate UI5 Tooling v2

### PR DESCRIPTION
**Pull request by d3xter666**
JIRA: CPOUI5FOUNDATION-661

This Pull request complements with:
- https://github.com/SAP/ui5-tooling/pull/828
-  https://github.com/SAP/ui5-cli/pull/631
- https://github.com/SAP/ui5-project/pull/613
- https://github.com/SAP/ui5-server/pull/597
- https://github.com/SAP/ui5-builder/pull/903
- https://github.com/SAP/ui5-fs/pull/506
- https://github.com/SAP/ui5-logger/pull/392

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,11 @@
 [![OpenUI5 Community Slack (#tooling channel)](https://img.shields.io/badge/slack-join-44cc11.svg)](https://join.slack.com/t/openui5/shared_invite/zt-1q128gn3p-JeZTi9XCpPxW8kBohSgqnw)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
-> There are two major versions of UI5 Tooling currently available:
-> - [UI5 Tooling v3](https://sap.github.io/ui5-tooling/v3)- Latest and stable version 🎉
-> - [UI5 Tooling v2](https://sap.github.io/ui5-tooling/v2)- In maintenance ⚠️
+
+> [UI5 Tooling v3](https://sap.github.io/ui5-tooling/v3) is the latest and stable version 🎉
+
+> [UI5 Tooling v2](https://sap.github.io/ui5-tooling/v2) has been deprectaed 🚫
```
Comment (flovogt) [inline README.md:13]:
```suggestion
> [UI5 Tooling v2](https://sap.github.io/ui5-tooling/v2) has been deprecated 🚫  
```

**Review by flovogt — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -7,9 +7,11 @@
...
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
-> There are two major versions of UI5 Tooling currently available:
-> - [UI5 Tooling v3](https://sap.github.io/ui5-tooling/v3)- Latest and stable version 🎉
-> - [UI5 Tooling v2](https://sap.github.io/ui5-tooling/v2)- In maintenance ⚠️
+
+> [UI5 Tooling v3](https://sap.github.io/ui5-tooling/v3) is the latest and stable version 🎉
+
+> [UI5 Tooling v2](https://sap.github.io/ui5-tooling/v2) has been deprecated 🚫  
+> Please migrate your projects to [UI5 Tooling v3](https://sap.github.io/ui5-tooling/v3/updates/migrate-v3/)!
```
Comment (flovogt) [inline README.md:14]: please align this PR with https://github.com/SAP/ui5-logger/pull/392/files

Comment (d3xter666) [reply README.md:14]: Done

**Review by flovogt — APPROVED** [review]

**Review by flovogt — CHANGES_REQUESTED** [review]
Screenshot 2023-05-25 at 13 13 26

Do we really wanna display this like it is?
I would recommend to remove the note syntax at all and use a plain text

**Comment by d3xter666** [comment]
> Screenshot 2023-05-25 at 13 13 26
> Do we really wanna display this like it is? I would recommend to remove the note syntax at all and use a plain text

I tought we should somehow highlight those and this seems to be the only way to do it.
Otherwise these announcements would be lost winthin the rest of the text.

What do you think about this formatting:
Screenshot 2023-05-25 at 15 23 10

**Review by matz3 — COMMENTED** [review]
LGTM

**Review by flovogt — APPROVED** [review]

**Pull request merged**